bool nmea_up
string nmea_topic
---
bool success